core/classes/Adapter.js

```diff
@@ -118,7 +118,34 @@ module.exports = class Adapter {
     return trx ? query.transacting(trx) : query;
   }
 
+  getKnexTypeMapping () { return ["integer", "bigInteger", "text", "string", "float", "decimal", "boolean", "date",
```

Comment:
1. I do not understand why this is a function – the contents is static, so you could do just something like this: `knexTypeMapping: ["integer", ...],`
1. Let's fix the naming. First, it doesn't have anything to do with knex – it's a postgres thing. Second, It's not a mapping – mapping expected to have some connections between entities. This array is just a list, so something like `postgresTypes`
1. Since it's a constant, you can move it outside of the module exports and declare just as `const POSTGRES_TYPES = ["integer", ...]` at the beginning of the module (constant naming used to be uppercased).